### PR TITLE
Feature: Drift info

### PR DESCRIPTION
**Pull request by hadeli**
Issues: #5

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -18,6 +18,11 @@ inputs:
     description: working directory
     required: true
     default: "."
+  terraform-plan-output:
```
Comment (shouze) [inline action.yaml:21]: won't you have to also introduce a plan-output for the drift part? Especially for cd prod/release needs? :thinking:

Comment (hadeli) [reply action.yaml:21]: Parent action define where plan should be stored

Comment (shouze) [reply action.yaml:21]: :+1: for the `plan`, but what about the `drift`?

Comment (hadeli) [reply action.yaml:21]: Parsed on message variable

**Review by shouze — APPROVED** [review]

**Pull request merged**